docs/design/design-patterns/non-functional-requirements-investigation-guide.md

```diff
@@ -0,0 +1,148 @@
+    * as necessary to achieve sufficient understanding of the probable impact of each requirement to the project
+    * both current/initial milestone and long-term/road map
+1. Document requirements/dependencies identified and related design constraints
+1. Evaluate current/near-term planned milestone(s) through the lens of the identified requirements/constraints
+    * Categorize each requirement as affecting immediate/near-term milestone(s) or as applicable instead to the longer-term road map/subsequent milestones
+1. Adapt plans for current/near-term milestone(s) to accommodate immediate/near-term-categorized requirements
+
+## Structure of Outline/Assignment of Responsible Stakeholder
+
+***
+In the following outline, assign name/email of 'responsible stakeholder' for each element after the appropriate level in the outline hierarchy.  Assume inheritance model of responsibility assignment: stakeholder at any ancestor (parent) level is also responsible for descendent (child) elements unless overridden at the descendent level).
+
+e.g.,
+
+* Parent1 *[Susan/susan@domain.com]*
+  * child1
+  * child2 *[John/john@domain.com]*
+    * grandchild1
+  * child3
+* Parent2 *[Sam/sam@domain.com]*
+  * child1
+  * child2
+
+In the preceding example, 'Susan' is responsible for `Parent1` and all of its descendants _except_ for `Parent1/child2` and `Parent1/child2/grandchild1`  (for which 'John' is the stakeholder).  'Sam' is responsible for the entirety of `Parent2` and all of its descendants.
+
+This approach permits the retention of the logical hierarchy of elements themselves while also flexibly interleaving the 'stakeholder' identifications within the hierarchy of topics if/when they may need to diverge due to e.g., CUSTOMER organizational nuances.
+
+***
+
+## Areas of Investigation
+
+* Enterprise Security
+  * Privacy
+    * PII
+    * HIPAA
+  * Encryption
+  * Data mobility
+    * at rest
+    * in motion
+    * in process/memory
+  * Key Management
+    * responsibility
+      * platform
+      * BYOK
+      * CMK
+  * INFOSEC regulations/standards
+    * e.g., FIPS-140-2
+      * Level 2
+      * Level 3
+    * ISO 27000 series
+    * NIST
+    * Other?
+  * Network security
+    * Physical/Logical traffic boundaries/flow topology
+      * Azure <-- --> On-prem
+      * Public <-- --> Azure
+      * VNET
+      * PIP
+      * Firewalls
+      * VPN
+      * ExpressRoute
+        * Topology
+        * Security
+    * Certificates
+      * Issuer
+        * CA
+        * Self-signed
+        * Rotation/expiry
+  * INFOSEC Incident Response
+    * Process
+    * People
+    * Responsibilities
+    * Systems
+    * Legal/Regulatory/Compliance
+* Enterprise AuthN/AuthZ
+  * Users
+  * Services
+  * Authorities/directories
+  * Mechanisms/handshakes
+    * Active Directory
+    * SAML
+    * OAuth
+    * Other?
+  * RBAC
+    * Perms inheritance model
+* Enterprise Monitoring/Operations
```

Comment:
I believe @zmmille2 was suggesting adding a link here to the observability section of the playbook

Reply:
Done.